src/PublicLab.Editor.js

```diff
@@ -48,6 +48,10 @@ PL.Editor = Class.extend({
 
       }
 
+      else {
```

Comment:
I have an optional suggestion.
Instead of if...else block
You can do
```js 
$('.ple-publish').prop('disabled', !condition);
```
Here, `condition` is the condition used for the if block. Just be sure to use the inverse using `!`

Reply:
I think you meant toggleClass instead of prop.
I have commited the changes using toggleClass instead of if...else block.
Nice Suggestion, Thanks!